docs/pages/button.md

```diff
@@ -68,11 +74,12 @@ Give a button additional meaning by adding a coloring class, or `.disabled` to c
 Add the `.hollow` class to a button to give it a hollow style. Change the `$button-fill` variable in your settings file to `hollow` to make this the default style. Changing this setting will remove the `.hollow` class from your CSS.
 
 ```html_example
-<button class="hollow button" href="#">Primary Color</button>
-<button class="secondary hollow button" href="#">Secondary Color</button>
-<button class="success hollow button" href="#">Success Color</button>
-<button class="alert hollow button" href="#">Alert Color</button>
-<button class="warning hollow button" href="#">Warning Color</button>
+<button class="hollow button" href="#">Primary</button>
+<button class="hollow button secondary" href="#">Secondary</button>
+<button class="hollow button success" href="#">Success</button>
+<button class="hollow button alert" href="#">Alert</button>
+<button class="hollow button warning" href="#">Warning</button>
+<button class="hollow button" href="#" disabled>Disabled</button>
```

Comment:
Oh don't get me wrong, I totally understand why you did it and you are right to encourage it. Just wondered if it would cause confusion thats all. Maybe where you mention about using the `.disabled` class you could also mention the attribute and keep this as is?